tools/scripts/setup-coral-credits.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+set -eux
+
+SITE=https://credits.apps.staging.hpc.cam.ac.uk
+
+TEST_PASSWORD="testpassword"
```

Comment:
I think we probably need a separate script that prompts the user for a password, then generates the token. I guess that is what the above script should be depending on.